### PR TITLE
Mark big downloads slow

### PR DESCRIPTION
**Pull request by sgugger**
This PR adds the slow decorator for models we don't want to download at each CI run.

### CODE REVIEW

File: tests/test_pipelines.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
 TEXT2TEXT_FINETUNED_MODELS = ["patrickvonplaten/t5-tiny-random"]
 TF_TEXT2TEXT_FINETUNED_MODELS = ["patrickvonplaten/t5-tiny-random"]
 
-DIALOGUE_FINETUNED_MODELS = ["microsoft/DialoGPT-medium"]
+DIALOGUE_FINETUNED_MODELS = ["microsoft/DialoGPT-medium"]  # @slow
```
Comment (LysandreJik) [inline tests/test_pipelines.py:34]: Why the comment here? Because it's a large model?

**Review by LysandreJik — APPROVED** [review]
LGTM, thanks!

File: tests/test_pipelines.py (hunk 1 above)
Comment (sgugger) [reply tests/test_pipelines.py:34]: I copied what was done above for `LARGE_FILL_MASK_FINETUNED_MODELS`

Comment (sgugger) [reply tests/test_pipelines.py:34]: (and yes it's a large model)

**Pull request merged**